public/js/acmeair-common.js

```diff
@@ -38,77 +38,63 @@ function hideWaitDialog() {
 }
 
 function updateLoggedInUserWelcome() {
-	var loggedinuser = dojo.cookie("loggedinuser");
-	if (loggedinuser == null) {
+	const session = dojo.cookie("acmeair-user");
+
+	if (!session) {
 		dojo.byId("loggedinwelcome").innerHTML = '';
-	}
-	else {
-		dojo.byId("loggedinwelcome").innerHTML = 'Welcome Back ' + loggedinuser;
+	} else {
+		const data = session.split(';')
+		dojo.byId("loggedinwelcome").innerHTML = 'Welcome Back ' + data[2];
```

Comment:
Incrementally, yes, I'd like to get the front and back end updated. The priority right now, however, is getting the backend updated since that's the part that Node.js' benchmarks actually look :-)